### PR TITLE
Cfoch/gh pages

### PR DESCRIPTION
**Pull request by cfoch**

### CODE REVIEW

File: releases/schedule.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ permalink: /releases/schedule/
 | [SDK3]        | February 5, 2018  |
 | **[SDK4]**    | May 4, 2018       | 3.28
 | [SDK5]        | August 2, 2018    |
-| SDK6          | November 1, 2018  | 3.30
+| SDK6          | To be announced   | 3.30
```
Comment (tchx84) [inline releases/schedule.md]: s/To be announced/T.B.D

File: releases/sdk5.md [hunk 2]
```diff
@@ -29,13 +29,11 @@ App developers are no longer limited to the SDK modules. Now, developers can wri
...
-- Runtime: VERSION file is no longer needed.
-- Modular Framework: Videos from Course apps can now be opened from global search.
-- Modular Framework: Copy option is now available for article images.
-- Modular Framework: Minor fixes to existing presets.
-- Renderer: Several improvements for showing articles in small screens.
+- Runtime: Update existing translations
+- Runtime: Fixed GPU acceleration for HTML content rendering for ARM architecture.
+- Modular Framework: Add translations for preset files.
+- Modular Framework: Cache compilation of scss files for applications.
+- Modular Framework: Allow to use extra gresources into an application.
```
Comment (tchx84) [inline releases/sdk5.md]: oh, you should left these here.

File: releases/sdk5.md [hunk 3]
```diff
@@ -47,3 +45,13 @@ No changes are required for apps using the modular framework.
...
+- Runtime: Unknown licenses will be displayed properly.
+- Runtime: Data model now supports `can_print` and `can_export` properties.
+- Runtime: VERSION file is no longer needed.
+- Modular Framework: Videos from Course apps can now be opened from global search.
+- Modular Framework: Copy option is now available for article images.
+- Modular Framework: Minor fixes to existing presets.
+- Renderer: Several improvements for showing articles in small screens.
+
+### SDK 5.1 (September 13, 2018) ###
+This is the last release in the SDK5 series.
```
Comment (tchx84) [inline releases/sdk5.md]: The older 5.x stay at the bottom, with their original comments, you only need to add a new entry with the newer changes for 5.1. Look at SDK-4 notes for examples.

File: releases/schedule.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ permalink: /releases/schedule/
 | [SDK3]        | February 5, 2018  |
 | **[SDK4]**    | May 4, 2018       | 3.28
 | [SDK5]        | August 2, 2018    |
```
Comment (ptomato) [inline releases/schedule.md]: Just a drive-by suggestion: you should un-bold SDK4 and bold SDK5, since it is the current version.
